src/components/Feed/Post.tsx

```diff
@@ -82,53 +83,39 @@ const ColorButton = withStyles((theme) => ({
   },
 }))(Button);
 
-export default function RecipeReviewCard() {
+export default function RecipeReviewCard(props: {recognition: any}) {
   const classes = useStyles();
-
+  const post = props.recognition;
   return (
     <Card className={classes.root}>
       <Box display="flex" flexDirection="row" style={{ marginTop: 8}}>
         <div>
           <img src={PlaceholderProfileImg} alt="profile" className={classes.profilePhoto} />
           <Typography variant="body2" color="textSecondary" style={{ marginLeft: 20 }}>
-              September 14, 2016
+              {(new Date(post.postDate)).toLocaleString('en-US', { month: 'long', day: 'numeric', year: 'numeric' })}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Benedict-Scrumberbatch/peer-recognition/pullRequest?prid=7204339)
```suggestion
              {(new Date(post.postDate)).toLocaleString("en-US", { month: 'long', day: 'numeric', year: 'numeric' })}
```